src/commands/install.ts

```diff
@@ -16,3 +22,29 @@ export const installCli = async () => {
 
   console.log(`\n✅ CLI installed. Run ${cyan('juno --help')} to display more information.`);
 };
+
+export async function installCliIfNecessary() {
```

Comment:
for consistency reason, can you use an arrow function or any particular reason to use a function here?